nestkernel/streamers.h

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
`libnestutil` would be a better location for this file.

Reply:
+1

Reply:
I agree